### PR TITLE
configmap changes for parallel queries

### PR DESCRIPTION
**Pull request by Poorna-BS-ML**

### CODE REVIEW

**Review by Sonika-Prakash-ML — APPROVED** [review]

File: charts/sf-archival/charts/query-controller/values.yaml [hunk 1]
```diff
@@ -36,6 +36,12 @@ db:
   queryexecutioncontroller:
     max_idle_connections: 1
     max_open_connections: 2
+parallelQuery:
+  queryController:
+    queryParallelismFactor: 3
```
Comment (Venunadh-Veeralanka-ML) [inline charts/sf-archival/charts/query-controller/values.yaml:41]: change it to 5

Comment (Poorna-BS-ML) [reply charts/sf-archival/charts/query-controller/values.yaml]: changed it to 5

File: charts/sf-archival/charts/query-controller/values.yaml [hunk 2]
```diff
@@ -36,6 +36,12 @@ db:
   queryexecutioncontroller:
     max_idle_connections: 1
     max_open_connections: 2
+parallelQuery:
+  queryController:
+    queryParallelismFactor: 5
+    steppedParallelism: false
```
Comment (Venunadh-Veeralanka-ML) [inline charts/sf-archival/charts/query-controller/values.yaml:42]: As mentioned if default value of  steppedParallelism is false then we need not give here.

Comment (Poorna-BS-ML) [reply charts/sf-archival/charts/query-controller/values.yaml:42]: as discussed we will not expose the steppedParallelism value in the overriding values file we use while upgrading

**Pull request merged**